pages/product.html

```diff
@@ -8,146 +8,135 @@
     <link rel="stylesheet" href="../src/input.css" />
     <title>Document</title>
   </head>
-  <body class="relative w-full h-full font-roboto">
-    <nav class="flex justify-between items-center py-2">
-      <div class="font-roboto-bold text-3xl">{finsweet</div>
-
-      <div class="flex gap-6 items-center">
-        <p class="color-primary font-roboto text-paragraph">
-          <a href="./home.html">Home</a>
-        </p>
-        <p class="color-primary font-roboto text-paragraph">
-          <a href="./product.html">Product</a>
-        </p>
-        <p class="color-primary font-roboto text-paragraph">
-          <a href="./pricing.html">Pricing</a>
-        </p>
-        <p class="color-primary font-roboto text-paragraph">
-          <a href="./aboutUs.html">About Us</a>
-        </p>
-        <p class="color-primary font-roboto text-paragraph">
-          <a href="./blog.html">Blog</a>
-        </p>
-        <p class="color-primary font-roboto text-paragraph">
-          <a href="./contact.html">Contact</a>
-        </p>
-        <p class="primary-btn">Free trial</p>
+  <body>
+    <nav class="nav">
+      <h1 class="nav__logo">{finsweet</h1>
+      <div class="nav__menu">
+        <p><a href="./home.html"> Home </a></p>
+        <p><a href="./product.html">Product</a></p>
+        <p><a href="./pricing.html"> Pricing</a></p>
+        <p><a href="./aboutUs.html"> About us </a></p>
+        <p><a href="./blog.html"> Blog </a></p>
+        <p><a href="./contact.html"> Contact</a></p>
+        <button class="primary-btn">Free trial</button>
       </div>
     </nav>
-    <div class="grid grid-cols-2 mt-20 mx-20 gap-10 my-20">
+    <div class="product-body">
```

Comment:
`product-body` is not meaningfull ... improve the naming, may be `product__hero`